opencensus/exporters/stats/stackdriver/internal/stackdriver_e2e_test.cc

```diff
@@ -225,7 +225,7 @@ TEST_F(StackdriverE2eTest, LargeTest) {
 
   // The number of values to record for each tag--total metric cardinality will
   // be tag_values^2.
-  const int tag_values = 10;
+  const int tag_values = 25;
```

Comment:
Why 25?

Reply:
So that the number of time series pushed exceeds the batch size. 15+ would work, but this is fast enough.

Reply:
IMO worth adding this as a comment. :)

Reply:
Done.